firebase-firestore/src/main/java/com/google/firebase/firestore/SetOptions.java

```diff
@@ -26,10 +26,10 @@
 
 /**
  * An options object that configures the behavior of {@code set()} calls. By providing one of the
- * SetOptions objects returned by {@link #merge}, {@link #mergeFields} and {@link #mergeFieldPaths},
- * the {@code set()} calls in {@link DocumentReference}, {@link WriteBatch} and {@link Transaction}
- * can be configured to perform granular merges instead of overwriting the target documents in their
- * entirety.
+ * SetOptions objects returned by {@link #merge()}, {@link #mergeFields(List<String>)} and {@link
```

Comment:
I don't think it's valid syntax. it should be something like:
```suggestion
 * SetOptions objects returned by {@link #merge()}, {@link #mergeFields(List)} and {@link
```

i.e. without generic parameters.

Reply:
Done.